cmd/sebak/cmd/wallet/payment.go

```diff
@@ -170,8 +169,16 @@ func init() {
 /// Returns:
 ///   `sebak.Transaction` = The generated `Transaction` creating the account
 ///
-func makeTransactionCreateAccount(kpSource keypair.KP, kpDest keypair.KP, amount common.Amount, seqid uint64, target string) transaction.Transaction {
-	opb := operation.NewCreateAccount(kpDest.Address(), amount, target)
+func makeTransactionCreateAccount(kpSource keypair.KP, kpDest keypair.KP, amount common.Amount, seqid uint64, isFrozen bool) transaction.Transaction {
+	var opb operation.CreateAccount
+	var fee common.Amount
+	if isFrozen {
+		opb = operation.NewCreateAccount(kpDest.Address(), amount, kpSource.Address())
+		fee = common.FrozenFee
+	} else {
+		opb = operation.NewCreateAccount(kpDest.Address(), amount, "")
+		fee = common.BaseFee
+	}
```

Comment:
That's not an improvement. What's wrong with the previous version ?

Reply:
IMO, To calculate fee, it is better to read

Reply:
Ok 👍